sdk/eventhub/Azure.Messaging.EventHubs/perf/Azure.Messaging.EventHubs.Perf/Azure.Messaging.EventHubs.Perf/Infrastructure/PerfTestEnvironment.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+
+namespace Azure.Messafging.EventHub.Perf
```

Comment:
```suggestion
namespace Azure.Messaging.EventHub.Perf
```

Reply:
Same typo in many places

Reply:
I missed this when reading over, but the namespace should be:
`Azure.Messaging.EventHubs.Perf`